docs/operator-guide/cd/deploy-rpm.md

```diff
@@ -0,0 +1,309 @@
+
+### GitOps Repository Structure
+
+Below is a structure of a GitOps repository designed for deploying RPM packages:
+
+```bash
+nexus.repo
+inventory.ini
+pre-deploy.yml
+├── web
+│   ├── dev
+│   │   ├── nano
+│   │   │   ├── 01_playbook.yml
+│   │   │   └── 02_playbook.yml
+│   │   └── atop
+│   │       ├── 01_playbook.yml
+│   │       └── 02_playbook.yml
+│   ├── qa
+│   │   ├── nano
+│   │   │   ├── 01_playbook.yml
+│   │   │   └── 02_playbook.yml
+│   │   └── atop
+│   │       ├── 01_playbook.yml
+│   │       └── 02_playbook.yml
+│   └── prod
+│       ├── nano
+│       │   ├── 01_playbook.yml
+│       │   └── 02_playbook.yml
+│       └── atop
+│           ├── 01_playbook.yml
+│           └── 02_playbook.yml
+└── db
+    ├── dev
+    │   ├── nano
+    │   │   ├── 01_playbook.yml
+    │   │   └── 02_playbook.yml
+    │   └── atop
+    │       ├── 01_playbook.yml
+    │       ├── 02_playbook.yml
+    │       └── 03_playbook.yml
+    ├── qa
+    │   ├── nano
+    │   │   ├── 01_playbook.yml
+    │   │   ├── 02_playbook.yml
+    │   └── atop
+    │       ├── 01_playbook.yml
+    │       ├── 02_playbook.yml
+    │       └── 03_playbook.yml
+    └── prod
+        ├── mysql
+        │   ├── 01_playbook.yml
+        │   └── 02_playbook.yml
+        └── cc
+            ├── 01_playbook.yml
+            ├── 02_playbook.yml
+            └── 03_playbook.yml
+```
+
+Below is a breakdown of the GitOps repository structure:
+
+- **nexus.repo**: Сontains the configuration for connecting to Nexus as a repository on the instances.
+- **inventory.ini**: Contains groups with instances, user credentials, and additional settings for connecting to the instances.
+- **pre-deploy.yml**: Contains general playbooks that contain a set of tasks that must be done before starting work, such as copying and connecting Nexus config, installing additional packages depending on the group.
+
+The rest of the files are example of deployment configurations. To spread these configuration files between Deployment Flows and Environments, files comply to the following hierarchy:
+
+```bash
+\<deployment-flow-name\>/\<environment-name\>/\<package-name\>/01_\<file\>.yml
+```
+
+For example, the **web/qa/nano/01_copy-file.yml** file relates to the **nano** application that belongs to the **qa** Environment, which in turn belongs to the **web** Deployment Flow. The file name must begin with its serial number (e.g., 01, 02, 03, ...), followed by an underscore, the rest of the file name, and the file extension.
+
+## AWX Approach
+
+The first approach to use RPM package is Default. This approach is considered more simple as it implies creating most of the resources via AWX user interface.
+
+### AWX Integration Flow
+
+Default approach involves the following steps:
+
+```mermaid
+graph LR;
+    A(Install AWX) --> B(Onboard Ansible Git repisitory) --> C(Configure AWX)--> D(Create application) --> E(Select deploy pipeline)
```

Comment:
A(Onboard Ansible Git repisitory) --> B(Install AWX)

Would this be more correct?